modules/social_features/social_group/social_group.module

```diff
@@ -235,23 +236,32 @@ function social_group_preprocess_group(array &$variables) {
   }
 
   // Add the hero styled image.
+  $group_image = '';
   if (!empty($group->get('field_group_image')->entity)) {
-    // Fetch image style from field info.
-    $original_image_style = $variables['content']['field_group_image'][0]['#image_style'] ?? '';
-    $image_style = ImageStyle::load($original_image_style);
-
-    if ($image_style instanceof ImageStyle) {
+    $group_image = $group->get('field_group_image')->entity;
+  }
+  else {
+    // Provide default image from field configuration.
+    $field_info = FieldConfig::loadByName($group->getEntityTypeId(), $group->bundle(), 'field_group_image');
```

Comment:
Consider the following here:
```suggestion
    $default_image = $group->get('field_group_image')->getSetting('default_image');
```

Then we don't need to import FieldConfig and load it by name separately. The field item should already have this information :)

Reply:
Comes from a comment here: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Field%21FieldDefinitionInterface.php/function/FieldDefinitionInterface%3A%3AgetDefaultValue/8.2.x